source/install/self-managed-changelog.md

```diff
@@ -4,12 +4,93 @@
 Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
 
 Latest Mattermost Releases:
-- [Release v6.2 - Feature Release](#release-v6.2-feature-release)
-- [Release v6.1 - Feature Release](#release-v6.1-feature-release)
-- [Release v6.0 - Feature Release](#release-v6.0-feature-release)
+- [Release v6.3 - Extended Support Release](#release-v6-3-extended-support-release)
+- [Release v6.2 - Feature Release](#release-v6-2-feature-release)
+- [Release v6.1 - Feature Release](#release-v6-1-feature-release)
+- [Release v6.0 - Feature Release](#release-v6-0-feature-release)
 - [Release v5.39 - Quality Release](#release-v5-39-quality-release)
 - [Release v5.37 - Extended Support Release](#release-v5-37-extended-support-release)
 
+## Release v6.3 - [Extended Support Release](https://docs.mattermost.com/upgrade/release-definitions.html#extended-support-release-esr)
+
+**Release Day: 2022-01-16**
+
+### Important Upgrade Notes
+ - An asynchronous job was added to fix an issue with Collapsed Reply Threads that resulted in unexpected unread channels. The job may result in an increase in server and database resource usage. Depending on the size of your Mattermost database, this job could take several hours to complete. In our internal testing it took about 3 hours to check and fix a ``ChannelMembership`` table with about 4 million rows in a PostgreSQL database.
```

Comment:
Do we have a sense on how many users does 4 million rows in ``ChannelMembership`` table equate to?

(From an admin's perspective, if I have 1,000 users in my system, should I expect more/less than 3 hours of downtime)?

Reply:
@amyblais let's update this with the new wording recommended here: https://github.com/mattermost/docs/pull/5344#discussion_r781660452

This is a good point that we should address if possible. ie we need to offer a conversion that admins might understand: ie # users, # channels, # posts are more relatable numbers than rows in the `ChannelMembership` table. If it's not possible to provide a direct conversion, perhaps we can offer a command or query that an admin can run to get the table size? @ashishbhate any thoughts ^